python_modules/libraries/dagster-databricks/dagster_databricks/configs.py

```diff
@@ -138,6 +138,16 @@ def _define_s3_storage_info():
     )
 
 
+def _define_aws_attributes_conf():
+    return Field(
+        Permissive(),
```

Comment:
ah for my original comment, I meant just substituting Shape(fields=...) for Permissive(fields=...). The Permissive object allows you to define some subset of possible fields, while allowing users to also include fields that are not included in that set.

this way, you get a bit of the best of both worlds -- you have stricter checking for the fields you know about, but in the future (if new attributes are added) you don't lock people out of using them. sorry for the confusion!

Reply:
Gotcha - updated again!